mlir/unittests/Analysis/PresburgerSetTest.cpp

```diff
@@ -591,4 +591,64 @@ TEST(SetTest, isEqual) {
   EXPECT_FALSE(rect.complement().isEqual(square.complement()));
 }
 
+void expectCoalesce(size_t expectedNumBasicSets, PresburgerSet set) {
+  PresburgerSet newSet = set.coalesce();
+  set.dump();
```

Comment:
We probably don't want to dump output for passing cases. If you want to keep this, I would suggest printing it only for failing cases.